### PR TITLE
Add support for Scala 2.13.7

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

**Comment by ckipp01** [comment]
In case someone looks here and is wondering about the status, we are needing this fixed https://github.com/scalacenter/bloop/issues/1598

File: mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala [hunk 1]
```diff
@@ -93,6 +93,7 @@ case class ScalaPresentationCompiler(
     workspace: Option[Path] = None
 ) extends PresentationCompiler:
 
+  //I should see name + age in autocomplete
```
Comment (ckipp01) [inline mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala:96]: Is this just leftover or can we put more context for what this is referring to?

**Review by ckipp01 — APPROVED** [review]
Just a small note on the comment, but other than that LGTM!

File: mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala (hunk 1 above)
Comment (tgodzik) [reply mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala:96]: leftover, I was testing out the other issue and forgot to remove part of the tested code :sweat:

**Pull request merged**